src/agents.py

```diff
@@ -12,7 +17,7 @@
 from datetime import datetime
 import json
 
-llm = ChatOpenAI(model="gpt-4o")
+llm = ChatOpenAI(model="gpt-4", api_key=os.getenv("OPENAI_API_KEY"))
```

Comment:
Why `gpt-4`?

Also, the api_key gets loaded automatically by `ChatOpenAI`

Reply:
This seems to be a mistake. I'll update